lib/src/utils/crypto/native.dart

```diff
@@ -13,7 +13,7 @@ abstract class Hash {
   FutureOr<Uint8List> call(Uint8List data) {
     final outSize = EVP_MD_size(ptr);
     final mem = malloc.call<Uint8>(outSize + data.length);
-    final dataMem = mem.elementAt(outSize);
+    final dataMem = mem + outSize;
```

Comment:
hm? 🤔

Reply:
```
  /// Pointer arithmetic (takes element size into account).
  @Deprecated('Use operator + instead')
  Pointer<Uint8> elementAt(int index) =>
      Pointer.fromAddress(address + sizeOf<Uint8>() * index);

  /// A pointer to the [offset]th [Uint8] after this one.
  ///
  /// Returns a pointer to the [Uint8] whose address is
  /// [offset] times the size of `Uint8` after the address of this pointer.
  /// That is `(this + offset).address == this.address + offset * sizeOf<Uint8>()`.
  ///
  /// Also `(this + offset).value` is equivalent to `this[offset]`,
  /// and similarly for setting.
  @Since('3.3')
  Pointer<Uint8> operator +(int offset) =>
      Pointer.fromAddress(address + sizeOf<Uint8>() * offset);
```